lib/stripe/api_operations/update.rb

```diff
@@ -25,16 +25,15 @@ def save(params={})
         # Now remove any parameters that look like object attributes.
         params = params.reject { |k, _| respond_to?(k) }
 
-        values = self.class.serialize_params(self).merge(params)
+        values = self.serialize_params(self).merge(params)
 
-        if values.length > 0
```

Comment:
I removed this because as far as I can tell the logic is just wrong. It may not be a necessarily successful request, but you should be able to post to a resource without any parameters.

There was even a test case for this already, and only a quirk in the old serialization logic was allowing it to succeed.